modules/ROOT/pages/digger/access-mobile-client-build-logs.adoc

```diff
@@ -0,0 +1,22 @@
+[[access-mobile-client-build-logs]]
+= Viewing Build Logs
+
+== Viewing Build History in the OpenShift Console
+
+All of the builds related to your mobile client can be seen in the **Builds** tab of your {mobile-client}. Each build can be expanded to show further information about the build configuration, latest build and build history.
+
+The latest build, as well as previous builds, has a `View Log` link associated with them to access the detailed logs of that build.
```

Comment:
All builds have a `View Log` link and clicking on it displays detailed log information.